app/helpers/reservation_helper.rb

```diff
@@ -14,21 +14,13 @@ def update_links(reservation)
     case reservation.status
     when Reservation::CREATED
       content_tag(:ul, class: 'p-0 d-flex mb-3') do
-        concat(button_to('Confirm', seance_reservation_path(
-                                      reservation.seance,
-                                      reservation,
-                                      status: Reservation::CONFIRMED
-                                    ),
+        concat(button_to('Confirm', confirm_reservation_path(reservation),
```

Comment:
I think you don't need to use [concat](https://apidock.com/rails/ActionView/Helpers/TextHelper/concat) :thinking: 
Have you spotted a problem here?

Reply:
When I remove concat only one button appears

Reply:
`concat` introduces a side effect in opposite to other helpers like `button_to` Read more about it [here](https://apidock.com/rails/ActionView/Helpers/TextHelper/concat), please. It could lead to surprising results in some cases if you for eg. will trigger this helper more times :wink:

Let's take a look at the idea behind the helper:
> A helper is a [method](https://www.rubyguides.com/2019/06/ruby-method-definition/) that is (mostly) used in your Rails views to share reusable code

I am almost sure that your helper won't be reusable, you put too many specific details in your helper to use it in other places of the app. You currently use it only in one place. So why do you want to complicate the view?